src/components/text-editor/text-editor.tsx

```diff
@@ -37,60 +13,84 @@ import { getFilteredMenu } from './menu/menu-filter';
     shadow: true,
     styleUrl: 'text-editor.scss',
 })
-export class TextEditor {
+export class TextEditor implements FormComponent<{ html: string }> {
```

Comment:
I'm not sure this is going to play nice if you want to place this component on a card etc. It would basically require you to write a wrapper component. I think we should change this to `FormComponent<string>` to make it easier to use

Reply:
Fixed in another PR, set to string and implicitly emits HTML. But there is still the question of whether we should emit HTML or markdown 🤔

Reply:
Ah, I got notified on so many PRs so maybe I didn't start in the right end :smile:

Reply:
Markdown supports HTML as well, but not the other way around, so I think markdown is better. That way you could have both and still only have one mode

Reply:
I will take care of this in a followup PR